cpp/src/lists/drop_list_duplicates.cu

```diff
@@ -50,89 +54,110 @@ struct has_negative_nans {
 };
 
 /**
- * @brief A structure to be used along with type_dispatcher to check if a
- * `column_view` has any negative NaN entry
+ * @brief A structure to be used along with type_dispatcher to check if a column has any
+ * negative NaN value.
  */
-struct has_negative_nans_fn {
+struct has_negative_nans_dispatch {
```

Comment:
Isn't it possible to have  `List<float>` coming through here that we'd want to check?

Reply:
No. This functor is called on the child column of the input lists column. If the input is a `List<List<float>>` column, exception will be thrown so the child column `List<float>` will never be passed here.